_posts/en/newsletters/2018-11-27-newsletter.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Bitcoin Optech Newsletter #23"
+permalink: /en/newsletters/2018/11/27/
+name: 2018-11-27-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter provides a reminder about potential feerate
+increases, summarizes suggested improvements to sighash flags to
+accompany BIP118 `SIGHASH_NOINPUT_UNSAFE`, and briefly describes a
+proposal to simplify fee bumping for LN commitment transactions.  Also
+included are selected recent Q&A from Bitcoin StackExchange and
+descriptions of notable code changes in popular Bitcoin infrastructure
+projects.
+
+## Action items
+
+- **Monitor feerates:** recent reductions in the exchange rate have lead
+  to modest decreases in hashrate and (possibly) increases in the number
+  of coins traveling to or from exchanges, which could lead to increased
+  feerates during the next week.  Unless there is a dramatic new change
+  in hashrate during the next week, a difficulty adjustment is expected
+  around Sunday that will mitigate most of the recent hashrate reductions.
+
+## News
+
+- **Sighash updates:** Pieter Wuille started a [thread][wuille sighash]
+  on the Bitcoin-Dev mailing list suggesting two additions for future
+  changes to segwit sighashes, especially [BIP118][]
+  `SIGHASH_NOINPUT_UNSAFE`.  A signature hash is the data committed to
+  by a signature in a transaction. Normally the hash commits to a list
+  of which coins are being spent, which scripts are receiving the coins,
+  and some metadata---but it's possible to sign only some of the
+  transaction fields in order to allow other users to change your
+  transactions in specific ways you might find acceptable (e.g. for
+  layer-two protocols).
+
+    Wuille suggests two additions to what metadata is hashed: first, the
+    transaction fee is included in the hash in order to ensure hardware
+    wallets or offline wallets can be absolutely sure that they aren't
```

Comment:
Style nit: Would changing from "absolutely sure" to "sure" change the meaning? :-)